instrumentation/ratpack/ratpack-1.7/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/ratpack/v1_7/RatpackTestSingletons.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.ratpack.v1_7;
+
+import io.netty.channel.ConnectTimeoutException;
+import java.net.URI;
+import org.junit.jupiter.api.condition.OS;
+import ratpack.http.client.HttpClientReadTimeoutException;
+
+public final class RatpackTestSingletons {
```

Comment:
since this class doesn't really contain any singletons it might be more appropriate to call it `RatpackTesUtil`